<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Leer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="proceedButton">
      <property name="text">
       <string>Weiter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="toggleButton">
      <property name="text">
       <string>Wechseln</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="outsideButton">
      <property name="text">
       <string>Au√üerhalb</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>Beenden</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
